scipy/spatial/transform/_rotation.pyx

```diff
@@ -1273,6 +1349,202 @@ cdef class Rotation:
         else:
             return cls(quat, normalize=False, copy=False)
 
+    @cython.embedsignature(True)
+    @classmethod
+    def from_davenport(cls, axes, order, angles, degrees=False):
+        """Initialize from Davenport angles.
+
+        Rotations in 3-D can be represented by a sequence of 3
+        rotations around a sequence of axes.
+
+        The three rotations can either be in a global frame of reference
+        (extrinsic) or in a body centred frame of reference (intrinsic), which
+        is attached to, and moves with, the object under rotation [1]_.
+
+        For Euler angles, we have the following conditions:
+
+            - ``dot(axis1, axis2) = 0``
+            - ``dot(axis2, axis3) = 0``
+            - ``dot(axis1, axis3) = -1``, ``0`` or ``1``
+
+        For Davenport angles, the third relation is relaxed [2]_:
+
+            - ``dot(axis1, axis3)`` can be any value between -1 and 1.
+
+        Parameters
+        ----------
+        axes : array_like
+            Specifies sequence of axes for rotations. Must be an array_like of
+            shape (3, ) or ([1 or 2 or 3], 3), where each axes[i, :] is the ith
```

Comment:
Better move into the parameter line after "array_like"?

Reply:
What do you mean? Sorry I didn't understand this one

Reply:
Ah, do you mean something like:
```python
"""
Parameters
----------
axes : array_like, shape (3,) or ([1 or 2 or 3], 3)
    Axis of rotation, if one dimentional. It two dimentional, describes the 
    sequence of axes for rotations, where each axes[i, :] is the ith
    axis. If more than one axis is given, then the second axis must be
    orthogonal to both the first and third axes.
"""
```

Reply:
Yes.

Reply:
Done!